state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -308,6 +328,12 @@ pub mod pallet {
 	#[pallet::getter(fn current_on_chain_key)]
 	pub type CurrentOnChainKey<T, I = ()> = StorageValue<_, AggKey<T, I>, OptionQuery>;
 
+	/// The current timeout duration for the broadcast, measured in number of blocks.
+	#[pallet::storage]
+	#[pallet::getter(fn broadcast_timeout)]
```

Comment:
these getters will be deprecated in the version we're about to upgrade to so would just remove it, it's not used rn too